requirements.txt

```diff
@@ -4,8 +4,6 @@ pipenv
 toml
 delegator.py
 pyyaml
-IGitt==0.4.1.dev20180722120744
-# Pinned due to igitt pinning.
```

Comment:
ogr is missing here.

Reply:
I though `requirements.txt` is obsolete :stuck_out_tongue: I will add it now!